source/src/cip/cipassembly.c

```diff
@@ -63,7 +63,7 @@ EipStatus CipAssemblyInitialize(void) {
 }
 
 void ShutdownAssemblies(void) {
-  CipClass *assembly_class = GetCipClass(kCipAssemblyClassCode);
+  CipClass *assembly_class = GetCipClass(kCipAssemblyClassId);
```

Comment:
same here, official name Class Code

Reply:
same here, ...

Reply:
Yeah. these are more markers for me, to not miss the needed changes. Eclipse is not the best tool for refactoring C code :)